DISCLAIMER.md

```diff
@@ -0,0 +1,27 @@
+# Disclaimer
+
+Simple Icons asks that users read this disclaimer fully before including an icon in their project.
+
+## Table of Contents
+
+* [Copyrights & Trademarks](#copyrights--trademarks)
+* [Guidelines](#guidelines)
+* [Removal of Brands](#removal-of-brands)
+
+## Copyrights & Trademarks
+
+The Simple Icons library contains a number of icons with specific requirements around their usage. Where this is the case, we aim to provide access to the proper guidelines through the `guidelines` entry in our JSON file & in our NPM package and through links on our [website](http://simpleicons.org/).
+
+We ask that users of our library read these guidelines fully and display the icons in accordance with them and, where applicable, with the necessary permissions.
+
+If a logo or icon includes a registered trademark or trademark symbol we follow the recommendations outline [in our Contribution Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#icon-guidelines) to decide whether to include the symbol or not.
+
+~Simple Icons cannot be held responsible for any legal activity raised by a brand over our users' usage of their icon.~
+
+## Guidelines
+
+Please note that the addition of guidelines to our library is an ongoing project so the absence of a `guidelines` entry for a particular icon does not imply that guidelines for that icon do not exist. Similarly, all guidelines are subject to change and, as we rely largely on our community to help us keep our library updated, the guidelines included in our library for a particular icon may be outdated. Therefore we also ask that users of our library do their own due diligence in researching a brand's guidelines.
```

Comment:
I would split the first part here as it is supposed to be temporary.

```suggestion
> NOTE: the addition of guidelines to our library is an ongoing project so the absence of a `guidelines` entry for a particular icon does not imply that guidelines for that icon do not exist.

As guidelines are subject to change and, as we rely largely on our community to help us keep our library updated, the guidelines included in our library for a particular icon may be outdated. Therefore we also ask that users of our library do their own due diligence in researching a brand's guidelines.
```

A second suggestion would be to request readers to inform us of updated brand guidelines, e.g. 

```suggestion
> NOTE: the addition of guidelines to our library is an ongoing project so the absence of a `guidelines` entry for a particular icon does not imply that guidelines for that icon do not exist.

As guidelines are subject to change and, as we rely largely on our community to help us keep our library updated, the guidelines included in our library for a particular icon may be outdated. Therefore we also ask that users of our library do their own due diligence in researching a brand's guidelines. Additionally, we ask you to inform us of outdated brand guidelines through the [GitHub repository].
```